packages/service-worker-mock/models/Request.js

```diff
@@ -44,6 +45,7 @@ class Request extends Body {
 
     this.method = options.method || 'GET';
     this.mode = options.mode || 'same-origin';   // FF defaults to cors
+    this.referrer = options.referrer || undefined;
```

Comment:
The spec says that if the referrer is `no-referrer` it should return an empty string. Can we change this line to
```
this.referrer = options.referrer && options.referrer !== 'no-referrer' ? options.referrer : '';
```

Reply:
done